### PR TITLE
1.0.0

### PR DESCRIPTION
**Pull request by Mackiovello**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+1. Clone this repository to your machine
+2. Install [Node.js](https://nodejs.org/en/)
+3. Download starpack.exe and starcounter.package.dll from [Dropbox](https://www.dropbox.com/sh/uq4eacrhm3fh0sn/AAA2vhuJL0cauTJx_ltha4MBa?dl=0) and put them in c:\programfiles\starcounter
```
Comment (warpech) [inline CONTRIBUTING.md]: Did you actually use starpack to create the build? Because `gruntfile.js` and `package.bat` use the old fashined way.

Comment (Mackiovello) [inline CONTRIBUTING.md]: Oh, I though it was used by grunt

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+1. Clone this repository to your machine
+2. Install [Node.js](https://nodejs.org/en/)
+3. Run npm install to get all dependencies
+4. Run `npm install grunt`
+5. Install grunt-contrib-watch, grunt-replace, grunt-bump, and grunt-shell by writing `npm install` and then the package
```
Comment (warpech) [inline CONTRIBUTING.md]: This is never done like that. Please add `packages.json` file that lists these dependencies. Then it is enough to write `npm install`.

**Comment by Mackiovello** [comment]
Decide to use starpack instead. This is therefore not relevant

**Pull request closed without merging**